Resources/Prototypes/_Sunrise/Loadouts/Miscellaneous/trinkets.yml

```
@@ -88,3 +88,16 @@
   storage:
     back:
     - ClothingNeckReedIndustriesPin
+
+# Wraps
+- type: loadout
+  id: ClothingWraps
+  storage:
+    back:
+    - ClothingWraps
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)